pkg/testing/integration/program.go

```diff
@@ -1325,6 +1361,67 @@ func (pt *ProgramTester) TestLifeCycleInitialize() error {
 		}
 	}
 
+	// Environments
+	for _, env := range pt.opts.CreateEnvironments {
+		name := pt.opts.getEnvNameWithOwner(env.Name)
+
+		initArgs := []string{"env", "init", name}
+		if err := pt.runPulumiCommand("pulumi-env-init", initArgs, dir, false); err != nil {
+			return err
+		}
+
+		envFile, err := func() (string, error) {
+			temp, err := os.CreateTemp("", "pulumi-env-test-*")
```

Comment:
Nothings cleaning these up? If you use the testing.T make temp file function then it will get auto-deleted after the test.

Reply:
Yeah, nothing's cleaning these up. Didn't feel that it was super important as these are tiny temp files. TIL about `testing.TempDir`--I'll use that.

Reply:
Fixed